entry_types/scrolled/package/src/frontend/PlayerControls/WaveformPlayerControls/Waveform.js

```diff
@@ -22,6 +22,8 @@ export function Waveform(props) {
             <div ref={measureRef} className={styles.waveWrapper}>
               <Wavesurfer mediaElt={`#${props.mediaElementId}`}
                           playing={props.isPlaying}
+                          play={props.play}
+                          pause={props.pause}
```

Comment:
I think this does not do anything. Originally setting the `playing` prop to true called play on Wavesurfer. Now it calls `playerActions.play`. But that's what caused `playerState.isPlaying` to become true in the first place. If I remove the `playing`/`isPlaying`, `play` and `pause` props here and in `WavefromPlayerControls/index.js` everything appears to work the same way. This makes sense since Wavesurfer already operates directly on the underlying media element.

This is also the way [it used to work in Paged](https://github.com/codevise/pageflow/blob/1f1a694151db27a0d5a535c279a0ffea78de72bb/entry_types/paged/packages/pageflow-paged-react/src/media/components/WaveformPlayerControls/Waveform.jsx#L25).

Reply:
You are right, I included this isPlaying property while trying to make waveform peaks visible previously, and let it remain there after the problem was fixed.